public/app/features/search/page/components/columns.tsx

```diff
@@ -225,11 +224,11 @@ function makeTypeColumn(
   styles: Record<string, string>
 ): TableColumn {
   return {
-    Cell: DefaultCell,
     id: `column-type`,
     field: kindField ?? typeField,
     Header: 'Type',
-    accessor: (row: any, i: number) => {
```

Comment:
using accessor only works for the first page.  Now that we are using the `InfiniteLoader`, we need to do this all in the `Cell` implementation